class-tgm-plugin-activation.php

```diff
@@ -2741,7 +2756,9 @@ protected function get_row_actions( $item ) {
 		 * @param object $item The current item.
 		 */
 		public function single_row( $item ) {
-			parent::single_row( $item );
+			echo '<tr class="' . esc_attr( 'tgmpa-type-' . strtolower( $item['type'] ) ) . '">';
+			parent::single_row_columns( $item );
```

Comment:
Please change this to `$this->single_row_columns( $item )` to allow for TGMPA potentially adding that method to the child class.